src/openms/source/TRANSFORMATIONS/FEATUREFINDER/EmgFitter1D.cpp

```diff
@@ -309,6 +309,69 @@ namespace OpenMS
     //width_ = weighted_sd * std::sqrt(std::pow(1. - (weighted_skew / 2.), 2./3.));
   }
 
+  void EmgFitter1D::setInitialParameters_(const RawDataArrayType& set)
+  {
+    if (param_.getValue("init_mom").toBool())
+    {
+      setInitialParametersMOM_(set);
+      return;
+    }
+
+    // sum over all intensities
+    CoordinateType sum = 0.0;
+    for (Size i = 0; i < set.size(); ++i)
+      sum += set[i].getIntensity();
```

Comment:
I think we either put it in same line (with/without {} ) or add brackets if we put it in next line